release notes/v0.36.0.md

```diff
@@ -0,0 +1,76 @@
+
+Thanks to the contribution of [@gernest](https://github.com/gernest), K6 now has the ability to abort a test. [#2093](https://github.com/grafana/k6/pull/2093) made the `k6/execution` exposes the `test.abort()` function, which allows to immediately abort a K6 load test run with the exit code `108`.
+
+Aborting is possible during initialization:
+```javascript
+import exec from "k6/execution";
+exec.test.abort();
+```
+
+As well as inside the default function:
+```javascript
+import exec from "k6/execution";
+
+export default function() {
+  // Note that you can abort with a specific message too
+  exec.test.abort("this is the reason");
+}
+
+export function teardown() {
+  console.log("teardown will still be called after test.abort()");
+}
+```
+
+Or during the VU initialization:
+```javascript
+import exec from "k6/execution";
+
+// This won't fail on initial parsing of the script, but on VU initialization.
+if (__VU == 1) {
+  exec.test.abort();
+}
+
+export default function() {}
+```
+
+### K6 inspect extended output ([#2279](https://github.com/grafana/k6/pull/2279))
+
+Following [#2279](https://github.com/grafana/k6/pull/2279), the `k6 inspect` command now supports an `--execution-requirements` flag. When used, the command's output will include fields related to the execution requirements, the configuration K6 derived from the execution context, and will also expose the `maxVUs` and `totalDuration` fields.
+
+
+### Forcing HTTP/1 protocol ([#2222](https://github.com/grafana/k6/pull/2222))
+
+Thanks to the work of [@sjordhani22](https://github.com/sjordhani22), [#2222](https://github.com/grafana/k6/pull/2222) made it possible to force K6 to use version 1.1 of the protocol when firing HTTP requests. 
+
+It can be done by setting the `http2client=0` value in the `GODEBUG` environment variable:
+
+```
+GODEBUG=http2client=0 k6 run testscript.js
+```
+
+**Nota Bene**: the usage of the `GODEBUG` variable is considered temporary, and expected to change in the future. If you start using this feature, keep an eye out for potential future changes.
+
+## Bugs Fixed!
+
+* We had a long-standing issue ([#1771](https://github.com/grafana/k6/issues/1771)) making opening large files in the context of a K6 script somewhat unreliable under specific conditions in the past. This has now been addressed by [#2314](https://github.com/grafana/k6/pull/2314).
+* We addressed an issue uncovered by [our community](https://community.k6.io/t/v0-35-0-grpc-server-reflection-error/2383), which kept our users using GRPC to define multiple services in a single *proto* file. This issue was addressed by [#2265](https://github.com/grafana/k6/pull/2265).
+* Thanks to the contribution of [@Resousse](https://github.com/Resousse), we've now updated K6's `go-ntlmssp` dependency. The updating PR [#2290](https://github.com/grafana/k6/pull/2290) indeed fixes issues with NTLM Authentication backends returning two authorization headers. 
```

Comment:
```suggestion
* Thanks to the contribution of @Resousse], we've now updated k6's `go-ntlmssp` dependency. The updating PR [#2290](https://github.com/grafana/k6/pull/2290) indeed fixes issues with NTLM Authentication backends returning two authorization headers. 
```

Reply:
I prefer to leave it as a plain Markdown link if that's okay with you. It both ensures that the link is present in the repo's markdown file itself. Also as a courtesy, as this might contribute to the search engine's ranking score of the person.

Reply:
I'm fine with it if the contributors' detection in the release page works fine with it. But I don't know how to check without releasing a new version. If you need an example, you can check the `contributors` section in [v0.35.0](https://github.com/grafana/k6/releases/tag/v0.35.0).

Reply:
I'm fine with it if the contributors' detection in the release page works fine with it. But I don't know how to check without releasing a new version. If you need an example, you can check the contributors section in v0.35.0.

Reply:
I must admit I didn't really understand what you meant with your follow up comment :slightly_smiling_face: The markdown file ends up being converted to HTML at the end of the pipeline. That's true whether Github does that on their page-generation step, or if we directly embed an actual Markdown HTML link in the markdown file. The content of the Markdown file will be copied verbatim in the release page when we do the release (I assume). So I think it should work as expected; assuming I understood your point correctly of course :bow:

Reply:
You mentioned in a private context:

> What I meant is that I'm not sure GitHub is able to populate the contributors section, e.g look at the end of the page https://github.com/grafana/k6/releases/tag/v0.35.0
4:11
> that section is automatically generated by github collecting all the mentions

I've looked into it and couldn't find any proper answer. Thus, I'll fall back to what you proposed which is the only documented format... 🤝